unicodetools/data/linkification/dev/LinkTerm.txt

```diff
@@ -0,0 +1,2156 @@
+# LinkTerm.txt
+# Date: 2025-12-14, 06:39:23 GMT 
+# © 2025 Unicode®, Inc.
+# Unicode and the Unicode Logo are registered trademarks of Unicode, Inc. in the U.S. and other countries.
+# For terms of use and license, see https://www.unicode.org/terms_of_use.html
+#
+# The usage and stability of these values is covered in https://www.unicode.org/reports/tr58/
+#
+# ================================================
+#
+# Property:	Link_Term
+# Format
+#
+# Field 0: code point range
+# Field 1: a Link_Term value
+#          For more information, see https://www.unicode.org/reports/tr58/#property-data. 
+#
+# For the purpose of detection and formatting operations, the property Link_Term is defined as
```

Comment:
We have been writing "For the purpose of regular expressions" as a hook for documenting names/types/values of non-UCD properties.